alacritty_terminal/src/grid/mod.rs

```diff
@@ -628,7 +649,7 @@ impl<T> BidirectionalIterator for GridIterator<'_, T> {
         match self.point {
             Point { column: Column(0), .. } => {
                 self.point.column = last_column;
-                self.point.line -= 1;
+                self.point.line -= Line(1);
```

Comment:
This isn't really an improvement, there's a reason why Add/SubAssign is implemented for both.